chatkit-core/src/test/kotlin/com/pusher/chatkit/users/UserApiTypeMapperTest.kt

```diff
@@ -0,0 +1,77 @@
+package com.pusher.chatkit.users
+
+import assertk.assertThat
+import assertk.assertions.isEqualTo
+import assertk.assertions.isNull
+import com.pusher.chatkit.users.api.UserApiType
+import com.pusher.chatkit.users.api.UserApiTypeMapper
+import com.pusher.chatkit.util.DateApiTypeMapper
+import org.spekframework.spek2.Spek
+import org.spekframework.spek2.style.specification.describe
+
+class UserApiTypeMapperTest : Spek({
+    describe("given a complete UserApiType object") {
```

Comment:
as per feedback shared on a previous PR lets consistently not use word "object" here as it does not add any helpful info

Reply:
👍 removed in both places.